diff --git a/init/main.c b/init/main.c
index 02f6b6bd6a17..59ebb6292ec6 100644
--- a/init/main.c
+++ b/init/main.c
@@ -1109,8 +1109,8 @@ static int __ref kernel_init(void *unused)
 	system_state = SYSTEM_RUNNING;
 	numa_default_policy();
 
-	printk(KERN_ERR "\n##### Natalie Poche (UFID: 1533-9442) Go Gators! #####\n\n");
+	printk(KERN_ERR "\n### Natalie Poche (Exercise 2) ###\n\n");

 	rcu_end_inkernel_boot();
 
 	if (ramdisk_execute_command) {
